client/network/src/lib.rs

```diff
@@ -253,7 +253,7 @@ mod discovery;
 mod light_client_handler;
 mod on_demand_layer;
 mod protocol;
-mod request_responses;
+pub mod request_responses;
```

Comment:
Shouldn't be public. I believe that everything you need is re-exported from the `config` module (which is public).